data/json/recipes/recipe_deconstruction.json

```diff
@@ -2043,32 +2043,32 @@
   {
     "result": "fork",
     "type": "uncraft",
-    "time": "1 m 30 s",
+    "time": "30 s",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "scrap", 1 ] ] ]
   },
   {
     "result": "glass",
     "type": "uncraft",
-    "time": "30 s",
+    "time": "1 s",
     "components": [ [ [ "glass_shard", 4 ] ] ]
   },
   {
     "result": "glass_bowl",
     "type": "uncraft",
-    "time": "30 s",
+    "time": "1 s",
```

Comment:
the glass bowl is 75ml smaller than the amount of glass shards listed here would suggest.

Reply:
Sure!